test/PanelGroupSpec.js

```diff
@@ -33,30 +34,27 @@ describe('<PanelGroup>', () => {
 
   it('Should not collapse panel by bubbling onSelect callback', () => {
```

Comment:
the changes to this test and the one below it, don't mirror what the original test did. In other words they aren't testing what the title says they are testing